lib/config/config-initializer.js

```diff
@@ -486,18 +489,35 @@ function promptUser() {
         // continue with the questions otherwise...
         return inquirer.prompt([
             {
-                type: "confirm",
-                name: "es6",
-                message: "Are you using ECMAScript 6 features?",
-                default: false
+                type: "list",
+                name: "ecmaVersion",
+                message: "Which version of ECMAScript do you use?",
+                choices: [
+                    { name: "ES3", value: 3 },
+                    { name: "ES5", value: 5 },
+                    { name: "ES2015", value: 2015 },
+                    { name: "ES2016", value: 2016 },
+                    { name: "ES2017", value: 2017 },
+                    { name: "ES2018", value: 2018 }
+                ],
+                default: 1 // This is the index in the choices list
             },
             {
                 type: "confirm",
                 name: "modules",
                 message: "Are you using ES6 modules?",
                 default: false,
                 when(answers) {
-                    return answers.es6 === true;
+                    return answers.ecmaVersion >= 2015;
+                }
+            },
+            {
+                type: "confirm",
+                name: "es6Globals",
+                message: "Are you using ES6 globals (such as `Map` and `Set`)?",
+                default: false,
+                when(answers) {
+                    return answers.ecmaVersion >= 2015;
```

Comment:
What's the reasoning for only asking this question when `answers.ecmaVersion >= 2015`? If we're giving users the option to enable ES2015 parsing without ES2015 globals, it seems equally plausible that they might want to enable ES2015 globals without ES2015 parsing.

To me, it seems better to either guess the globals option based on the `ecmaVersion` answer (keeping things simpler for the user), or to always ask both questions. I would slightly prefer the former, since the `--init` helper is just supposed to address common cases and doesn't need to provide the same level of flexibility as creating a config file manually.

Reply:
So if the user choose "greater or equals to ES2015", skip this question and enable ES2015 globals?

Reply:
Yes, that would be my recommendation.

Reply:
Should I remove this question? I think, if the user choose ES3 or ES5, it shouldn't enable ES2015 globals, and if the user choose "greater or equals to ES2015", enable it by default.

Reply:
Agreed, I think it would be best to remove the question and do what you just described.